tests/shared/app/controllers/forms_controller.rb

```diff
@@ -12,6 +12,9 @@ def form1
   def form2
   end
 
+  def form3
+  end
+
```

Comment:
IMHO we should be more descriptive with the name of the form fixtures. By just numbering we will eventually loose track of what each fixture is used for.

Maybe `form_with_two_inputs_in_separate_css_blocks_with_the_same_selector`? 
WDYT?

Reply:
Yes, that's a good idea. I will change that.

The other thing is, that the step is not so efficient as it iterates over all matching containers and calls `find` on each of them. This will result in `n * Capybara.wait_time` when none of them has a matching input field. But I don't have another solution.